src/createStore.js

```diff
@@ -24,14 +24,31 @@ export default (
   return {
     get(key) {
       return pipe(
-        key => get(namespace + key),
+        key => {
+          try {
+            return get(namespace + key)
+          } catch (e) {
+            if (!onGetError) throw e
+
+            return onGetError(e, key)
```

Comment:
I can include it, the reason I didn't include it is because the `namespace` is an option, while the error occurs (probably) when you get a specific key. Does that make sense?

Reply:
It does 👍